xgi/classes/reportviews.py

```diff
@@ -497,3 +533,17 @@ def members(self, e=None, dtype=list):
             raise IDNotFound(f'ID "{e}" not in this view')
 
         return self._id_dict[e].copy()
+
+    def singletons(self):
```

Comment:
would it be worth it to make this a property?

Reply:
I'm leaning toward keeping it as a method. Reason being that properties are expected to be fast and it's unclear to me how fast this is. Also, some users might expect a property to be set-able and this one definitely won't be.